examples/agent_builder/auto-discussion.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""A simple example for auto discussion: the agent builder automatically\
+ set up the agents participating the discussion ."""
+from utils import load_txt, extract_scenario_and_participants
+import agentscope
+from agentscope.agents import DialogAgent
+from agentscope.pipelines.functional import sequentialpipeline
+
+model_configs = [
+    {
+        "type": "openai",
```

Comment:
Please update keys "type", "name" into "config_name", "model_type" and "model_name" as [README.md](https://github.com/modelscope/agentscope/blob/main/README.md#openai-api-config)